apps/frontend/src/i18n/messages/en.ts

```diff
@@ -20,7 +20,13 @@ export default {
   'project.contributors': 'Contributors',
   'project.projectUpdated': 'Updated at',
   'sideBar.createProject': 'Start new project:',
+  'layerManager.fileformat':
```

Comment:
You should use `camelCase` for the key and change it where it appears.
Same goes for the keys below ....